accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/helper/AcHelper.java

```diff
@@ -135,7 +136,7 @@ private static Principal getPrincipalForName(final Session session, String princ
         // Also see https://issues.apache.org/jira/browse/OAK-3228
         final JackrabbitSession js = (JackrabbitSession) session;
         final UserManager userManager = js.getUserManager();
-        final Authorizable authorizable = userManager.getAuthorizable(principalName);
+        final Authorizable authorizable = userManager.getAuthorizable(new PrincipalImpl(principalName));

```

Comment:
Is this better, because the name is actually the id, so the lookup is much faster?